ocaml/sdk-gen/java/main.ml

```diff
@@ -288,6 +288,8 @@ let gen_method file cls message params async_version =
 
   fprintf file "    /**\n" ;
   fprintf file "     * %s\n" (escape_xml message.msg_doc) ;
+  fprintf file "     * Minimum allowed role: %s\n" (get_minimum_allowed_role
```

Comment:
Could not find a better option for [javadoc](https://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html) style docs. So it's just in the generic method summary